src/loader/bgw_launcher.c

```diff
@@ -321,13 +322,25 @@ start_db_schedulers(HTAB *db_htab)
 
 		worker_registered = register_entrypoint_for_db(current_entry->db_oid, vxid, &current_entry->db_scheduler_handle);
 		if (!worker_registered)
-			report_error_on_worker_register_failure();
-
+		{
+			hash_seq_term(&hash_seq);
+			ereport(LOG, (errcode(ERRCODE_INSUFFICIENT_RESOURCES),
+						  errmsg("TimescaleDB background worker scheduler for at least one database unable to start."),
+						  errhint("some background worker schedulers have been started, others have not. Increase max_worker_processes and restart the server.")));
```

Comment:
Should we report how many workers were/should-have-been started in this hint: `errhint("%d background worker schedulers out of %d have been started. Increase max_worker_processes and restart the server.", nstarted, ndatabases)` to assist the user in setting the correct number of bgw?